.aws/src/config/index.ts

```diff
@@ -43,6 +43,7 @@ export const config = {
     sqsPermLibItemMainQueueName: `PermLib-${environment}-ItemMain`,
     unifiedEventStreamName: 'unified_event',
     databaseTz: 'US/Central',
+    eventBusName: `PocketEventBridge-${environment}-Shared-Event-Bus`,
```

Comment:
let's make this 'default' so we don't have to change it later?

Reply:
done